src/minimax/minimax_rpa.F

```diff
@@ -10111,4 +10111,6526 @@ SUBROUTINE get_fit_coef_k20(Rc, L_b, U_b, fit_coef)
 
    END SUBROUTINE get_fit_coef_k20
 
+! **************************************************************************************************
+!> \brief ...
+!> \param k ...
+!> \param E_range ...
+!> \param aw ...
+! **************************************************************************************************
+   SUBROUTINE get_rpa_minimax_coeff_larger_grid(k, E_range, aw)
+      INTEGER                                            :: k
```

Comment:
Please add intents.

Reply:
Can you please explain what exactly you are meaning? All your other points below are clear to me and will be addressed.

Reply:
You have added intents in minimax_exp_gw.F, but not here. (INTENT(IN), INTENT(OUT), INTENT(INOUT))

Reply:
oh, I see, I will add, thanks!